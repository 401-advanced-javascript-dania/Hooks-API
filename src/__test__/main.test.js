import React from 'react';
import { shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../component/header.js';
import Footer from '../component/footer.js';
describe('<Header/>',()=>{
  it('exists at the start of our app',()=>{
    let app = shallow(<Header/>);
    expect(app.find('h1').exists()).toBeTruthy();
  });
  it('render correctly',() =>{
    const tree = renderer.create(<Header/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
describe('<Footer/>',()=>{
  it('exists at the start of our app',()=>{
    let app = shallow(<Footer/>);
    expect(app.find('footer').exists()).toBeTruthy();
  });
  it('render correctly',() =>{
    const tree = renderer.create(<Footer/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});