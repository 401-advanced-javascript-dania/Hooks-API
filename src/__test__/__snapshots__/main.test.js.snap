// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> render correctly 1`] = `
<footer>
  done by dania :-D
</footer>
`;

exports[`<Header/> render correctly 1`] = `
<header>
  <h1>
    My TO Do List App
  </h1>
</header>
`;
